### DISCUSSION TITLE
My btop UI shows weird character

### DISCUSSION THREAD

**Discussion by hejunchen** [aristocratos/btop#757] [Q&A]
I use btop frequently, but I have noticed my btop screen uses weird character for the progress.

This is from my ubuntu box (physical pc):
![image](https://github.com/aristocratos/btop/assets/6847102/a93a249c-4970-45b9-9684-717161582115)

This is from my PVE LXC (linux container):
![image](https://github.com/aristocratos/btop/assets/6847102/3ee81a7e-b835-4bcf-bbf0-c9d5444d476c)

------------------------------------------------------------

Below is the the official screenshot from this Github repo. Notice the character used are different. The official version is definitely better. So what is wrong with my btop? Any idea?

<img src="https://raw.githubusercontent.com/aristocratos/btop/main/Img/normal.png" />

**Comment by iambumblehead** [comment]
+1 just installed btop on a new system and noticed the same thing --the progress bar visual is not so refined-looking. Maybe we lack a system font with braille characters https://github.com/aristocratos/btop/issues/783

_edit_ noticed that, using another terminal (gnome's default terminal), the braille fonts were rendered and the progress bar looks detailed again. Changing terminal is another option...
